packages/atomic/src/components/atomic-query-summary/atomic-query-summary.tsx

```diff
@@ -66,6 +67,10 @@ export class AtomicQuerySummary implements InitializableComponent {
     return `<span class="font-bold" part="highlight">${content}</span>`;
   }
 
+  private wrapQuery(query: string) {
+    return `<span title="${query}" class="font-bold truncate" part="highlight query">${query}</span>`;
```

Comment:
Make sure to test with a screen reader (or multiple) that this `title` doesn't reduce accessibility.